src/main/resources/i18n/Resources_nl.properties

```diff
@@ -16,40 +16,27 @@ folderSelectorDirectoryChooserTitle=Selecteer de media folder
 mainScreenAboutDialogTitle=Over
 mainScreenInitialUploadMethodDialogTitle=You've got a choice!
 preferencesDialogTitle=Voorkeuren
-#TODO translate
-preferencesDialogTabPatterns=Files
-#TODO translate
-preferencesDialogTabPatternsLinkText=Click here for documentation on pattern format
-#TODO translate
-preferencesDialogTabPatternsIgnoreDescription=If a file matches any of these patterns, it will be ignored
+preferencesDialogTabPatterns=Bestanden
+preferencesDialogTabPatternsLinkText=Klik hier voor documentatie over de mogelijkheden
+preferencesDialogTabPatternsIgnoreDescription=Als een bestand voldoet aan een regel, dan wordt deze genegeerd
 preferencesDialogIgnoreSectionTitle=Negeer deze bestanden en of folders
-#TODO translate
-preferencesDialogTabMain=Main
-#TODO translate
-preferencesDialogIncludeSectionTitle=Only include these files or folders
-#TODO translate
-preferencesDialogTabIncludeDescription=If this list is not empty, then only files matching any of these patterns will be uploaded, except the files that match \
-  any of the ignore patterns
-#TODO translate
-preferencesUploaderStrategyLabel=Due to the limitations of Google Photos API, only one of the following is possible. Please make your choice.
-#TODO translate
-preferencesDialogUploaderStrategySectionTitle=Upload Method
-#TODO translate
-preferencesUploaderStrategyPreferDuplicateSupportLabel=Some of the files I will upload are already uploaded by other apps or directly.
-#TODO translate
-preferencesUploaderStrategyPreferDuplicateSupportDescription=Media items uploaded by other apps or directly will be matched and added to the appropriate \
-  album, which is good. However, media items will be sorted within the album only by their file name, and not by the date they are taken. \
-  They will have to be sorted later manually, if needed.
-#TODO translate
-preferencesUploaderStrategyPreferOrderLabel=I do not care about photos already uploaded by other apps or directly, but prefer items auto-ordered by date \
-  within albums.
-#TODO translate
-preferencesUploaderStrategyPreferOrderDescription=Uploaded media items will be sorted within the album by the date they are taken, which is good. However, \
-  items uploaded by other apps or directly will not be added to the appropriate album and will display an error next to them. Such items will have to be \
-  added to their albums later manually, if needed.
-#TODO translate
-preferencesLoadingFatalError=Unable to start: Failed loading preferences. Are you trying to run the earlier version after the later version has been run? \
-  This is not supported. Please run the latest version.
+preferencesDialogTabMain=Hoofd
+preferencesDialogIncludeSectionTitle=Voeg alleen deze bestanden en mappen toe
+preferencesDialogTabIncludeDescription=Als deze lijst niet leeg is, worden alleen bestanden geüpload die overeenkomen met een van deze patronen, behalve de bestanden die overeenkomen met \
```

Comment:
Can I ask you to reformat these a little? The line width must be no longer than 160 characters, and the backslash is a way to split the line in Java .properties files. Thanks!

Reply:
Sorry, I meant to do it for all of the properties that you added, so that there are no lines in the file longer than 160